packages/libs/src/sdk/api/challenges/ChallengesApi.ts

```diff
@@ -163,7 +168,7 @@ export class ChallengesApi extends BaseAPI {
     )
 
     logger.debug('Disbursing claim...')
-    const disbursement = await this.evaluateAttestations({
```

Comment:
does disbursement have anything else that we're no longer returning?

Reply:
`disbursement` was a string. all i did here was rename and return in object. the thing that was being returned is still attainable to the caller